lib/blobs/usbboot/config.txt

```diff
@@ -0,0 +1,6 @@
+kernel=kernel7.img
+gpu_mem=16
+enable_uart=1
```

Comment:
...and if we're getting rid of the `console=` in cmdline.txt, we can get rid of this enable_uart line too ;)

Reply:
Done!